platform/commonUI/edit/res/templates/edit-action-buttons.html

```diff
@@ -20,11 +20,30 @@
  at runtime from the About dialog for additional information.
 -->
 <span ng-controller="EditActionController">
-    <span ng-repeat="currentAction in editActions">
+    <!-- If there's a single save action show a button, otherwise show a dropdown with all save actions. -->
+    <span ng-if="saveActions.length === 1">
+        <a class='s-button major {{saveActions[0].getMetadata().cssclass}}'
+           title='{{saveActions[0].getMetadata().name}}'
+           ng-click="saveActions[0].perform()">
+            <span class="title-label">{{saveActions[0].getMetadata().name}}</span>
+        </a>
+    </span>
+
+    <span ng-if="saveActions.length > 1">
+        <mct-control key="'menu-button'"
+                     structure="{
+                        options: saveActionsAsMenuOptions,
+                        click: saveActionMenuClickHandler,
+                        cssclass: 'btn-bar right icon-save no-label major'
+                     }">
+        </mct-control>
+    </span>
+
+    <span ng-repeat="currentAction in otherEditActions">
         <a class='s-button {{currentAction.getMetadata().cssclass}}'
```

Comment:
`CancelAction`'s CSS includes `no-label` but I have to pass `metadata.name` as text for each action, and `mct-button` thinks the control needs a label, so it adds `labeled` to its CSS. A label doesn't show but the cancel button appears slightly wider than normal. It cannot be unseen.

I can solve this by making CancelAction's name `undefined`. Since the action appears to only be relevant in the edit bar, I don't think this will result in any control missing a label saying "Cancel". It'd raise the eyebrows of anyone reading the code though.

I'm leaning towards leaving the `otherEditActions` as they were, especially since it gets rid of the nasty `cssclass: ($index === 0 && saveActions.length === 0 ? 'major ' : '') + currentAction.getMetadata().cssclass`. Which makes the first `otherEditActions` button major.